.github/workflows/pypi.yml

```diff
@@ -51,3 +51,5 @@ jobs:
       with:
         user: __token__
         password: ${{ secrets.PYPI_ACCESS_TOKEN }}
+    permissions:
+      actions: read
```

Comment:
We should be using trusted publishers here.

See https://docs.pypi.org/trusted-publishers/adding-a-publisher/